tests/test_tokenize.py

```diff
@@ -33,8 +33,13 @@ def __init__(self, x, y):
         self.y = y
 
 
+def serialize(value, path):
```

Comment:
super nit: Since this is a top-level function, maybe a slightly more specific name, like `serialize_to_path` or something?

Reply:
Sure thing, just updated.